dev/com.ibm.ws.microprofile.graphql.1.0_fat/publish/servers/mpGraphQL10.basicMutation/bootstrap.properties

```diff
@@ -1,4 +1,4 @@
-com.ibm.ws.logging.trace.specification=*=info:LogService=all:com.ibm.ws.microprofile.graphql.*=all
+com.ibm.ws.logging.trace.specification=*=info:LogService=all:com.ibm.ws.microprofile.graphql.*=all:com.ibm.ws.io.smallrye.graphql.*=all:io.smallrye.graphql.*=all
```

Comment:
do these traces currently need to be on? we should only enable them if we suspect there is an issue that we need to reproduce in order to limit how much we trace in the builds.